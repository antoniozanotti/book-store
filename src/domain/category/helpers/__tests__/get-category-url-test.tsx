import { getCategoryUrl } from "../get-category-url";
import { categoriesWithPageMock } from "../../__tests__/categories-with-page-mock";

describe("getCategoryUrl", () => {
  it("should return category url", async () => {
    const url = getCategoryUrl(categoriesWithPageMock[0]);
    expect(url).toMatchSnapshot();
  });
});
