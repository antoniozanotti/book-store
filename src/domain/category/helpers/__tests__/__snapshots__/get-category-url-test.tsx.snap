// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getCategoryUrl should return category url 1`] = `"/category/arts-and-entertainment"`;
